### PR TITLE
Refresh table synopsis

### PR DESCRIPTION
**Pull request by mikethebeer**

### CODE REVIEW

File: docs/sql/refresh.txt [hunk 1]
```diff
@@ -8,10 +8,37 @@ Crate is `eventually consistent`_. Data written with a former statement is
...
+    ...   content string,
+    ...   width double,
+    ...   day timestamp
+    ... ) CLUSTERED BY (title) INTO 4 SHARDS PARTITIONED BY (day);
+    CREATE OK (... sec)
+
+.. hide:
+
+    cr> INSERT INTO parted_table (id, title, width, day)
+    ... VALUES (1, 'Don''t Panic', 19.5, '2014-04-08');
```
Comment (chaudum) [inline docs/sql/refresh.txt]: 'Don\''t'

**Comment by chaudum** [comment]
ack. squash and merge

File: docs/sql/refresh.txt [hunk 2]
```diff
@@ -8,10 +8,37 @@ Crate is `eventually consistent`_. Data written with a former statement is
...
+    INSERT OK, 1 row affected (... sec)
+
+In case the ``PARTITION`` clause is omitted all partitions will be refreshed. ::
+    
+    cr> REFRESH TABLE locations, parted_table;
```
Comment (mfussenegger) [inline docs/sql/refresh.txt]: imho having multiple tables in the example is just missleading

File: docs/sql/refresh.txt [hunk 3]
```diff
@@ -8,10 +8,37 @@ Crate is `eventually consistent`_. Data written with a former statement is
...
+    REFRESH OK (... sec)
+
+
+In case of using the ``PARTITION`` clause only specific partitions of a partitioned 
+table are refreshed. ::
+
+    cr> REFRESH TABLE locations,
```
Comment (mfussenegger) [inline docs/sql/refresh.txt]: same here

**Pull request merged**